cs/v1/CS_v1.0.0.yaml

```diff
@@ -904,123 +912,96 @@ components:
             - PLN (Planned)
         eventDateTime:
           type: string
-          x-stoplight:
-            id: 8o5gse4ovp1ah
           format: date-time
           description: 'The local date and time, when the event takes place, in ISO 8601 format specifying the offset.'
           example: '2025-01-14T09:21:00+01:00'
+    ServiceSchedule:
+      title: Service Schedule
+      type: object
+      required:
+        - carrierServiceName
+        - carrierServiceCode
+        - vesselSchedules
+      properties:
+        carrierServiceName:
+          type: string
+          description: The name of the service.
+          maxLength: 50
+          pattern: ^\S(?:.*\S)?$
+          example: Great Lion Service
+        carrierServiceCode:
+          type: string
+          maxLength: 11
+          example: FE1
+          pattern: ^\S(?:.*\S)?$
+          description: The carrier-specific code of the service for which the schedule details are published.
+        universalServiceReference:
+          type: string
+          pattern: ^SR\d{5}[A-Z]$
+          maxLength: 8
+          minLength: 8
+          example: SR12345A
+          description: 'A global unique service reference, as per DCSA standard, agreed by VSA partners for the service.'
+        vesselSchedules:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/VesselSchedule'
     VesselSchedule:
```

Comment:
Are there no 'voyages' in this object?

Reply:
It's similar to the OVS so the voyages are inside the transportCall